src/sentry/static/sentry/app/views/eventsV2/tags.tsx

```diff
@@ -163,21 +163,16 @@ class Tags extends React.Component<Props, State> {
 
   render() {
     return (
-      <TagSection>
-        <StyledHeading>{t('Event Tag Summary')}</StyledHeading>
+      <React.Fragment>
+        <StyledSectionHeading>{t('Event Tag Summary')}</StyledSectionHeading>
         {this.renderBody()}
-      </TagSection>
+      </React.Fragment>
     );
   }
 }
 
-// These styled components are used in getsentry for a paywall.
```

Comment:
Is this no longer the case?

Reply:
No. The getsentry upsell uses request stubbing now. This removed the need to rebuild the page layout in the upsell state.